process_csv.php

```diff
@@ -180,7 +185,15 @@ function process_selected_organization(string $selectedorgsourcedid, string $tem
         set_config('datasync_schools', $selectedorgsourcedid, 'enrol_oneroster');
     }
 
-    $csvclient->set_data($manifest, $users, $classes, $orgs, $enrollments, $academicsessions);
+
+    $version = get_config('enrol_oneroster', 'oneroster_version');
+    if ($version == ' Version 1.2'){
```

Comment:
Again this version check has == and a space. Can we clean up the strings from the options perhaps?